packages/rest/src/index.ts

```diff
@@ -8,6 +8,7 @@ export * from './router';
 export * from './providers';
 
 export * from './parser';
+export * from './body-parsers';
```

Comment:
I have mixed feelings about using plural form here, perhaps `./body-parser` or `./body-parsing` would be better? Just an idea to consider, I am not sure what's the best name.

Reply:
☝️